addons/web/static/tests/views/list/list_view.test.js

```diff
@@ -194,22 +199,52 @@ class Currency extends models.Model {
 
 defineModels([Foo, Bar, Currency, ResCompany, ResPartner, ResUsers]);
 
+async function clickRecordSelector() {
+    if (getMockEnv().isSmall) {
+        await contains("tbody td.o_data_cell:eq(0)").drag();
+    } else {
+        await contains(`tbody tr.o_data_row[data-id] input`).click();
```

Comment:
`[data-id]` not necessary

Reply:
I would use `tbody .o_data_row .o_list_record_selector input` to be more explicit